src/dashboards/Map/index.tsx

```diff
@@ -7,26 +7,55 @@ import {
     useWalkTime,
 } from '../../logic'
 
+import MapView from '../../components/Map'
+
 import './styles.scss'
-import MapView from './MapView'
+import { useSettingsContext } from '../../settings'
+import { DEFAULT_ZOOM } from '../../constants'
 
+import DepartureTag from './DepartureTag'
 const MapDashboard = ({ history }: Props): JSX.Element => {
+    const [settings] = useSettingsContext()
+
     const stopPlacesWithDepartures = useStopPlacesWithDepartures()
     const bikeRentalStations = useBikeRentalStations()
     const walkTimes = useWalkTime(stopPlacesWithDepartures)
-
+    const HEADER_MARGIN = 16
```

Comment:
VET IKKE OM DU TRENGER CAPS LOCK HER? ER SMÅPIRK ALTSÅ, MEN STILL 😅

Reply:
Tror det er konvensjonen for faktiske konstanter 😺

Reply:
https://media.giphy.com/media/wMvESGxZ0Cqd2/giphy.gif